packages/editor/src/client/panel/publishArticlePanel.tsx

```diff
@@ -13,26 +13,33 @@ import {InfoColor} from '../atoms/infoMessage'
 
 export interface PublishArticlePanelProps {
   initialPublishDate?: Date
+  initialUpdateDate?: Date
+  availableFromDate?: Date
   pendingPublishDate?: Date
   metadata: ArticleMetadata
 
   onClose(): void
-  onConfirm(publishDate: Date, updateDate: Date): void
+  onConfirm(publishDate: Date, updateDate: Date, availableOnlineFrom: Date): void
 }
 
 export function PublishArticlePanel({
   initialPublishDate,
+  initialUpdateDate,
+  availableFromDate,
   pendingPublishDate,
   metadata,
   onClose,
   onConfirm
 }: PublishArticlePanelProps) {
   const now = new Date()
 
-  const [publishDate, setPublishDate] = useState<Date | undefined>(
-    pendingPublishDate ?? initialPublishDate ?? now
+  const [publishDate, setPublishDate] = useState<Date | undefined>(initialPublishDate ?? now)
+
+  const [availableOnlineFrom, setAvailableOnlineFrom] = useState<Date | undefined>(
```

Comment:
I recommend to use `publishAt` and `setPublishAt` even though we translate it with "Available Online From". The field is called `publishAt` throughout the project and it makes it much easier to identify for us developers.